### PR TITLE
Feature/other councils

### PR DESCRIPTION
**Pull request by Adamws33**
Updated logo to be dynamic, made selector landing page and cannot access explorer till state is selected.  removed map for midwest. removed soil info for midwest, added goals API

### CODE REVIEW

File: src/pages/GoalsSelector/GoalsSelector.js [hunk 1]
```diff
@@ -46,34 +59,17 @@ const GoalsSelector = () => {
...
-                <div className="col-3">
-                  <Skeleton style={goalSkeletonStyle} />
-                </div>
-              </div>
-            </div>
-          ) : (
+          {allGoals.length > 0 && (
             <Grid container spacing={4} className="goals" style={{ justifyContent: 'center' }}>
-              {allGoals[0].fields ? (
+              {allGoals.length > 0 ? (
```
Comment (mikahpinegar) [inline src/pages/GoalsSelector/GoalsSelector.js:64]: this is guaranteed to be true by line 62 reight?

File: src/store/Reducer.js [hunk 2]
```diff
@@ -109,6 +109,41 @@ const Reducer = (state, action, value = action && action.data && action.data.val
       };
     }
 
+    case 'UPDATE_STATE': {
+      if (action.data.council === 'NECCC') {
+        return {
+          ...state, state: action.data.state, council: 'Northeast', stateId: action.data.stateId,
```
Comment (mikahpinegar) [inline src/store/Reducer.js:115]: seems like maybe the long name "Western" vs "wccc" should also be returned by the api so this doesn't need to be hard coded here

**Review by mikahpinegar — CHANGES_REQUESTED** [review]

File: src/pages/GoalsSelector/GoalsSelector.js (hunk 1 above)
Comment (Adamws33) [reply src/pages/GoalsSelector/GoalsSelector.js:64]: good catch!

**Comment by mikahpinegar** [comment]
the text for the state dropdown being light green makes it hard to read

**Comment by mikahpinegar** [comment]
selecting a southern state shows no goals and gives this error 

```
GoalsSelector.js:25 Uncaught (in promise) TypeError: state.councilId is not iterable
    at getAllGoals (GoalsSelector.js:25:1)
    at GoalsSelector.js:47:1
    at commitHookEffectListMount (react-dom.development.js:23150:1)
    at commitPassiveMountOnFiber (react-dom.development.js:24926:1)
    at commitPassiveMountEffects_complete (react-dom.development.js:24891:1)
    at commitPassiveMountEffects_begin (react-dom.development.js:24878:1)
    at commitPassiveMountEffects (react-dom.development.js:24866:1)
    at flushPassiveEffectsImpl (react-dom.development.js:27039:1)
    at flushPassiveEffects (react-dom.development.js:26984:1)
    at performSyncWorkOnRoot (react-dom.development.js:26076:1)
```

**Pull request closed without merging**